components/sup/src/manager/service_updater.rs

```diff
@@ -37,7 +38,7 @@ const DEFAULT_FREQUENCY: i64 = MIN_ALLOWED_FREQUENCY;
 type UpdaterStateList = HashMap<ServiceGroup, UpdaterState>;
 
 enum UpdaterState {
-    AtOnce(Receiver<PackageInstall>),
+    AtOnce(Receiver<PackageInstall>, Sender<bool>),
```

Comment:
It's not immediately obvious to me what the `bool` here represents. Perhaps an `enum` would make it clearer?

This goes for the other `Sender<bool>`s as well.

Reply:
I'm all for saving bytes.  And yes, @christophermaier is right that I just wanted to send a signal.  The value is irrelevant.  I'll change it to `()`